libs/core/src/components/popover/popover.ts

```diff
@@ -109,6 +111,42 @@ export class GdsPopover extends GdsElement {
   @property({ attribute: false })
   calcMaxHeight = (_referenceEl: HTMLElement) => `500px`
 
+  /**
+   * Whether the popover is nonmodal. When true, the popover will not trap focus and other elements
+   * on the page will still be interactable while the popover is open.
+   */
+  @property({ type: Boolean })
+  nonmodal = false
+
+  /**
+   * When this is set to `true`, the `:backdrop` pseudo-element will be visible if the popover is
+   * in modal mode. When not in modal mode (using the `nonmodal` attribute), this can instead be
+   * set to a selector matching a `<gds-backdrop>` element, in wich case the popover will take
+   * control of that backdrop.
+   *
+   * Example:
+   * ```html
+   * <gds-popover backdrop=".my-backdrop">
```

Comment:
This example here is rendering for some reason on the Public API table 

![CleanShot 2024-07-09 at 10 48 53@2x](https://github.com/seb-oss/green/assets/2398447/f4bcfe7d-42ce-438b-b170-ed8bbdae8fda)

Reply:
Yes, saw that lol! Gonna look into how to escape stuff properly for Storybook.

Reply:
Had just missed to close the code fence block :rage4: